atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -516,6 +536,95 @@ private void unmarkStreamsAsUsed() {
                     line("index.delete(toDelete);");
                 } line("}");
             }
+
+            private void storeStreamWithCompression() {
+                line("@Override");
+                line("public Pair<", StreamId, ", Sha256Hash> storeStream(InputStream stream) {"); {
+                    line("long id = storeEmptyMetadata();");
+                    line();
+                    line("MessageDigest digest = Sha256Hash.getMessageDigest();");
```

Comment:
Duplicated (generation) code here. Consider extracting the repeated group of `line()`s, or better yet, generate a private helper method.